backend/src/main/java/gov/cdc/usds/simplereport/db/model/DeviceType.java

```diff
@@ -26,6 +27,8 @@
   @Column(nullable = false)
   private String swabType;
 
+  @Transient private String swabTypes;
```

Comment:
added this to satisfy graphql type, this is temporary and will be used when we fully support multiple swab types